src/components/DefineCompany/AddNameTranslation.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div id="name-translation">
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" ref="ntForm" name="business-contact-form">
+      <v-row>
+        <v-col class="pb-0">
+          <v-text-field
+            filled
+            persistent-hint
+            label="Enter Name Translation"
+            v-model="nameTranslation"
+            :rules="nameTranslationRules">
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col class="form__btns pt-0">
+          <v-btn large color="error" id='btn-remove'
+            :disabled="!editNameTranslation"
+            @click="removeTranslation">
+              Remove
+          </v-btn>
+          <v-btn large color="primary" id='btn-done' class="form-primary-btn"
+            :disabled="!nameTranslationForm"
+            @click="addTranslation">
+              Done
+          </v-btn>
+          <v-btn large class="form-cancel-btn" id='btn-cancel'
+            @click="cancelTranslation">
+              Cancel
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class NameTranslation extends Vue {
+  @Prop({ default: '' })
+  private editNameTranslation: string
+
+  // Local Properties
+  private nameTranslationForm: boolean = false
+  private nameTranslation: string = ''
+
+  // Validation Rules
+  private readonly nameTranslationRules: Array<Function> = [
+    v => !!v || 'A name is required', // is not empty
+    v => /^[a-z|0-9A-Z]+(?: [a-z|0-9A-Z]+)*$/.test(v) || 'Invalid character', // numbers, letters and single spaces only
```

Comment:
Does this allow French characters? Also I think numbers are not allowed

Reply:
I'll look into this, looking at design details now..

Reply:
Yes to French!
Actually wants to allow special characters, but not numbers as well. 
Will update in next commit!

Reply:
I don't think this regex works for French characters. (I tested it with Séverin at https://regex101.com/).

Ref: http://www.asciitable.com/

Reply:
You bet, this is an outdated commit. It has been updated in the latest to allow French chars :)